### ISSUE TITLE
Missing Ability to Zoom

### ISSUE THREAD

**Issue by hermitdash** [RoyalBingBong/meView#16]
Topic Title more or less.

Kinda a critical feature for image viewing. Videos not so much because you could always full screen it. But when images are just too large and you wanna go in deeper to see details a Zoom would be nice.

Really like the simplicity of the app tho, good work!

**Comment by RoyalBingBong** [comment]
I agree zooming would be great, but it would also be a hell to implement. 😖  
The current view works purely with CSS, which I though was pretty nice. But zooming with the scrollwheel or with the usual shortcuts like <kbd>+</kbd> or <kbd>-</kbd> would require some additional magic.

I'll see what I can come up with. Though, it will probably take a while.

**Comment by Atulin** [comment]
I think at least a simple `fit to view`/`show full res` switch would do for now.
